File: to-do-app/To-Do-Java/src/main/java/com/renan/todo/dto/UserDTO.java
package com.renan.todo.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * User data to return as login operations
 */
@NoArgsConstructor
@AllArgsConstructor
@Data
public class UserDTO {

    private String  email;
    private String  jwt;
    private Integer id;
    private Integer status;
    private Integer expiresIn;
    private String  name;
    private String  lastName;

}

File: webstore/webstore-go/src/controller/dto/userDto.go
package dto

// UserDTO struct with data to use as response for login operations
type UserDTO struct {
	Email     string   `json:"email"`
	JWT       string   `json:"jwt"`
	Status    uint64   `json:"status"`
	ExpiresIn uint64   `json:"expiresIn"`
	Name      string   `json:"name"`
	LastName  string   `json:"lastName"`
	Roles     []string `json:"roles"`
}
